quadriga/src/main/webapp/WEB-INF/views/layouts/main.html

```diff
@@ -50,6 +50,14 @@
 	          		<a th:href="@{/admin/user/list}" >Users</a>
 	          	</li>
 	          	
+	          	<li sec:authorize="hasRole('USER')" role="presentation">
+	          		<a th:href="@{/auth/collections/add}" >Add Collection</a>
+	          	</li>
+	          	
+	          	<li sec:authorize="hasRole('USER')" role="presentation">
+	          		<a th:href="@{/auth/collections/showcollection}" >Show Collections</a>
+	          	</li>
```

Comment:
this should be a drop down menu "Collections" with two menu items (See all and Add collection)